pkgs/dartpad_ui/lib/enable_gen_ai.dart

```diff
@@ -5,6 +5,12 @@
 // Turn on or off gen-ai features in the client.
 const bool genAiEnabled = true;
 
+// Set to true to use GenUI instead of Gemini.
+// This is a temporary flag to test GenUI in production.
+// It will be removed once GenUI is fully integrated.
+// The flag is set based on URL parameter in main.dart.
+bool useGenUI = false;
```

Comment:
It's a little bit weird that we're modifying this global from a random spot. Can this maybe be:

`late bool useGenUI;`

So if it is attempted to be accessed before `main.dart` sets it, the app crashes?